server/client_fake.go

```diff
@@ -25,6 +25,12 @@ func (client *clientFake) NodeRegister(proposal dto_discovery.ServiceProposal) (
 	return nil
 }
 
+func (client *clientFake) RegisterIdentity(identity dto_discovery.Identity) (err error) {
+	log.Info(MYSTERIUM_API_LOG_PREFIX, "Identity registered: ", identity)
```

Comment:
Mention something in logs, that fake registration was used e.g. `Fake identity registered`